src/edu/stanford/dlss/was/WasapiDownloader.java

```diff
@@ -9,12 +9,19 @@
   public WasapiDownloaderSettings settings = null;
 
 
-  public WasapiDownloader(String settingsFileLocation, String[] args) throws IOException {
+  public WasapiDownloader(String settingsFileLocation, String[] args) throws LoadSettingsException {
     settings = new WasapiDownloaderSettings(settingsFileLocation, args);
   }
 
+  public void run() {
```

Comment:
The name run() feels a little...awkward...here. When I see a run() method in Java, I automatically assume that the current class is meant to run multithreaded. Would it be appropriate to rename the method to execute() or something like that?